src/App.js

```diff
@@ -1,24 +1,14 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
-
+import Sidebar from './components/Sidebar/Sidebar';
```

Comment:
@morelmiles this file is going to break our current codebase for the features in place

Reply:
the way around it is to first rebase with develop branch, then also import this component in the routes.js file